bitswap/README.md

```diff
@@ -15,56 +15,87 @@ Reviewers:
 
 # Abstract
 
-Bitswap is the data trading module for ipfs, it manages requesting and sending blocks to and from other peers in the network. Bitswap has two main jobs, the first is to acquire blocks requested by the client from the network. The second is to judiciously send blocks in its posession to other peers who want them.
-
+Bitswap is the data trading module for ipfs, it manages requesting and sending
+blocks to and from other peers in the network. Bitswap has two main jobs, the
```

Comment:
Keeping it one line is fine :)

Reply:
Gotcha, I'll change that back. Kind of unfortunately painful to deal with for my text editor, but I'll get by :)

Reply:
@dgrisham which editor are you using?

Reply:
[Kakoune](https://github.com/mawww/kakoune) -- it has visual soft-wrapping, but unfortunately it still behaves like a single line when you use that feature. It's a young text editor, so these things come with the territory. I just made a Bash function that uses `pandoc` to convert the input file to one with 80-character lines, then once I exit the text editor it converts it back so that the paragraphs are all on one line again (as it is here).